VirtoCommerce.Storefront/Controllers/Api/ApiCartController.cs

```diff
@@ -167,6 +167,77 @@ public class ApiCartController : StorefrontControllerBase
 
         }
 
+
```

Comment:
Could we leave only one (bulk) AddItemsToCart method, and remove old which used for single item addition?

Reply:
yes we can. I fix it